src/components/interaction-designer/block-types/Core_LogBlock.vue

```diff
@@ -23,6 +23,7 @@
             :placeholder="'flow-builder.enter-message' | trans"
             :current-expression="value"
             :valid-state="isValid"
+            data-cy="log-block--message--input"
```

Comment:
any particular reason we prefixed with `log-block--` ?

Reply:
You're right, it was not necessary. Renamed!